src/VirtualGrid.vue

```diff
@@ -394,6 +397,15 @@ export default class VirtualGrid<P> extends Vue {
     getElementOffset(element: Element) {
         return window.scrollY + element.getBoundingClientRect().top;
     }
+    getListenerTarget() {
```

Comment:
Can you provide an alternative version of the example that would be impacted by this change?

Reply:
It's not uncommon to see this design pattern of scrolling divs, rather than scrolling the whole page. For example, spotify  https://open.spotify.com/show/3IM0lmZxpFAY7CwMuv9H4g This page actually scrolls the div, not the whole page

Reply:
oh sorry, i meant to add to the current example a version with a scrolling div, unfortunately i didn't had enough time to add proper testing to the project, so the demo is only proof it works well  :s

Reply:
I should be able to find time to make the relevant changes this week.
Sorry, English is not my native language and I misunderstood you.

Reply:
Me neither, no worries, ok sure, take your time :)